backend/src/Manager/CompanyInfo/CompanyInfoManager.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Manager\CompanyInfo;
+
+use App\AutoMapping;
+use App\Constant\CompanyInfo\CompanyInfoConstant;
+use App\Entity\CompanyInfoEntity;
+use App\Repository\CompanyInfoEntityRepository;
+use App\Request\CompanyInfo\CompanyInfoCreateRequest;
+use App\Request\CompanyInfo\CompanyInfoUpdateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+
+class CompanyInfoManager
+{
+    private AutoMapping $autoMapping;
+    private EntityManagerInterface $entityManager;
+    private CompanyInfoEntityRepository $companyInfoEntityRepository;
+
+    public function __construct(AutoMapping $autoMapping, EntityManagerInterface $entityManager, CompanyInfoEntityRepository $companyInfoEntityRepository)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->entityManager = $entityManager;
+        $this->companyInfoEntityRepository = $companyInfoEntityRepository;
+    }
+
+    public function create(CompanyInfoCreateRequest $request)
+    {
+        $companyInfoAll = $this->getCompanyInfo();
+
+        if (! $companyInfoAll) {
+            $companyInfoEntity = $this->autoMapping->map(CompanyInfoCreateRequest::class, CompanyInfoEntity::class, $request);
+
+            $this->entityManager->persist($companyInfoEntity);
+            $this->entityManager->flush();
+
+            return $companyInfoEntity;
+        }
+        else {
```

Comment:
no need for (else) here:
 if (! $companyInfoAll) {
          ...
            return $companyInfoEntity;
        }
return CompanyInfoConstant::COMPANY_INFO_EXISTS;